### PR TITLE
codegen: v1 GA release

### PR DESCRIPTION
**Pull request by jkwlui**
**Do not merge**

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -75,5 +76,12 @@ module.exports = gapic.v1beta1;
  */
 module.exports.v1beta1 = gapic.v1beta1;
 
+/**
```
Comment (crwilcox) [inline src/index.js]: With the release of v1, do we want the default to be v1? (line 70)

File: CONTRIBUTORS [hunk 2]
```diff
@@ -5,6 +5,6 @@
 #
 Alexander Fenster <fenster@google.com>
 Chris Wilcox <crwilcox@google.com>
-Christopher Wilcox <git@crwilcox.com>
+Jonathan Lui <kinwa91@msn.com>
```
Comment (alexander-fenster) [inline CONTRIBUTORS]: can we use Google email here (and on GitHub as well)?  Can be done by adding you to `.mailmap` and regenerating scaffolding.

File: src/index.js (hunk 1 above)
Comment (alexander-fenster) [reply src/index.js]: Yes, absolutely!

Comment (alexander-fenster) [reply src/index.js]: @crwilcox should `synthtool` take care of that as well?

File: CONTRIBUTORS (hunk 2 above)
Comment (kinwa91) [reply CONTRIBUTORS]: Yes please! I just updated my primary email on Github. Where shall I edit this `.mailmap`

**Review by alexander-fenster — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (crwilcox) [reply src/index.js]: @alexander-fenster one of the next things we are adding is the ability to merge these index files. You can do it today with replacement patterns but it isn't elegant.

File: CONTRIBUTORS (hunk 2 above)
Comment (crwilcox) [reply CONTRIBUTORS]: Looks like my one above was my google :) Is it pretty standard to change the default github and .git config to be @google addresses?

**Comment by JustinBeckwith** [comment]
@kinwa91 we ready to merge here?

**Comment by kinwa91** [comment]
I was asked not to release this yet, will merge once that changes.

**Pull request merged**